_data/beyond-sidebar.yml

```diff
@@ -497,6 +503,11 @@
             - title: Update shipping address
               link: https://api-docs.beyondshop.cloud/ng-order-order-public-api.html#resources-order-update-shipping-address
               id: update_shipping_address
+        - title: Order batch actions
+          entries:
+            - title: Show order batch action status
```

Comment:
As discussed - to avoid confusion why the other order batch actions are not listed here, they should all be listed in the explanation of this endpoint.

Reply:
As discussed we leave it the way it was due to susceptibility to errors